components/resources/BUILD.gn

```diff
@@ -89,11 +89,16 @@ repack("resources") {
     ]
   }
 
-  if (brave_wallet_enabled && !is_android) {
-    deps += [ "//brave/components/brave_wallet_ui:resources" ]
-    sources += [
-      "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak",
-    ]
+  if (brave_wallet_enabled) {
+    if (!is_android) {
+      deps += [ "//brave/components/brave_wallet_ui:resources" ]
+      sources += [
+        "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak",
```

Comment:
now that we'll have 2 different pak files though I should come back and rename this to `brave_wallet_ui.pak`